pages/common/pr.md

```diff
@@ -0,0 +1,28 @@
+# pr
+
+> Paginate or columnate files for printing.
+> More information: <https://www.gnu.org/software/coreutils/manual/html_node/pr-invocation.html>.
+
+- Print multiple files with a default header and footer:
+
+`pr {{file1 file2 file3}}`
```

Comment:
I'd be tempted to use `{{file1}} {{file2}} {{file3}}` instead of `{{file1 file2 file3}}` here.

Reply:
Reads better and I think that's how we deal with this in the majority of the pages.

Reply:
I used the tar page as reference for mutliple files input, but I agree that it would look nicer with separated tokens.

Reply:
Ah, interesting @monst12! Perhaps we should update the `tar` page then.

Reply:
Sure, go ahead! A PR would be great @monst12 :D